bundles/org.openhab.ui/web/src/pages/settings/things/inbox/inbox-list.vue

```diff
@@ -3,7 +3,7 @@
     <f7-navbar title="Inbox" back-link="Things" back-link-url="/settings/things/" back-link-force>
       <f7-nav-right>
         <f7-link icon-md="material:done_all" @click="toggleCheck()"
-                 :text="(!$theme.md) ? ((showCheckboxes) ? 'Done' : 'Select') : ''" />
+                 :text="(!$theme.md) ? ((showCheckboxes) ? 'Done' : 'MultiSelect') : ''" />
```

Comment:
Why this change?

Reply:
In my opinion it makes it more clear what to expect. 
When using the new ui the first time I didn´t even realized that this was a clickable Item. You can click on every entry by its own to select it. 
The renaming to 'Multiselect' is more intuitive if you are looking for an option to select more than one list-item.

Reply:
Yes but MultiSelect is not a word. It was a deliberate choice to name this link like that since iPhone users would recognize it from the Photos app on their phone, among others (I'm using Android primarily too but I know for a fact that the market share of Apple products among OH users is almost 50% so we have to account for that):

![image](https://user-images.githubusercontent.com/2004147/121578710-3fef8b80-ca2b-11eb-941a-85cc01c14a95.png)

Reply:
```suggestion
                 :text="(!$theme.md) ? ((showCheckboxes) ? 'Done' : 'Select') : ''" />
```